snorkel/annotations.py

```diff
@@ -367,10 +375,12 @@ def load_matrix(matrix_class, annotation_key_class, annotation_class, session,
     # Cycles through the entire table to load the data.
     # Perfornamce may slow down based on table size; however, negligible since 
     # it takes 8min to go throuh 245M rows (pretty fast).
+    for res in session.execute(annot_select_query):
+        # NOTE: The order of return seems to be switched in Python 3???
+        # Either way, make sure the order is set here explicitly!
+        cid, kid, val = res.candidate_id, res.key_id, res.value
 
-    for val, cid, kid in session.execute(annot_select_query):
-
-        if cid in cid_to_row and kid in kid_to_col:
+        if (cid in cid_to_row) and (kid in kid_to_col):
```

Comment:
The parentheses in this if statement are not needed as `in` is higher [precedence](https://docs.python.org/3/reference/expressions.html#operator-precedence) than `and`.